### PR TITLE
Dem changes

### PR DESCRIPTION
**Pull request by talogan**
Update RTC Product Guide to reflect addition of the Copernicus DEM

### CODE REVIEW

File: docs/guides/rtc_product_guide.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ When custom-ordering imagery, however, the DEM Matching option is available for
 
 During processing, a surface scattering area image for the scene is calculated and saved. This projected area image is used to create the RTC product - the SAR image is multiplied by the ratio of an ellipsoidal scattering image (used during calibration) and this scattering area image. Note that this image is, by default, projected to gamma-nought (γ<sub>0</sub>). However, when custom ordering imagery,  selecting sigma-nought (σ<sub>0</sub>) output will result in a sigma-nought projected area image.
```
Comment (hjkristenson) [inline docs/guides/rtc_product_guide.md:97]: @talogan I thought that the scattering image was sigma-0 by default, but this description has the opposite. Thought I'd check to make sure.

File: docs/guides/rtc_product_guide.md [hunk 2]
```diff
@@ -181,11 +246,7 @@ This scale is not always the best choice for general visualization of RTC produc
...
-
-The two satellites that comprise the Sentinel-1 mission each have a 12-day repeat cycle, so most areas of the earth will have imagery at least every 6 days, but many areas have coverage even more frequently, making this SAR dataset a very useful tool for monitoring rapid or sudden landscape changes. In addition, SAR is not impacted by either cloud cover or lack of light, so RTC imagery can be collected at any time, and in areas or situations where cloud cover often causes problems for other imagery types.
+The RTC products are presented as Cloud-Optimized GeoTIFFs (COGs), a user-friendly format that is GIS compatible. The products include pre-generated overviews, so users will not need to generate pyramids to display the images efficiently in a GIS environment.
 
 The following sections present examples of how one might use RTC datasets to identify areas of change and integrate RTC datasets into other datasets for enhanced results. We also present a bibliography of some of the scientific literature making use of Sentinel-1 RTC datasets.
```
Comment (hjkristenson) [inline docs/guides/rtc_product_guide.md:251]: Add a blurb pointing to the story maps

File: docs/guides/rtc_product_guide.md (hunk 1 above)
Comment (talogan) [reply docs/guides/rtc_product_guide.md]: I checked the code and my emails.  The scattering area image is actually gamma0.  There is no sigma0 scattering area image calculated right now.  

Instead, GAMMA's pixel_area program creates
- **gamma0_pix** (scattering area image saved and the simulated image used during co-registration)
- **sigma_ratio image** (which converts ellipsoidal radiometrically calibrated sigma0 into DEM-based sigma0)
- **sig2gam_ratio image** (which converts ellipsoidal radiometrically calibrated sigma0 into DEM-based  gamma0)

During processing, the appropriate one of these ratio images is multiplied by the sigma0 calibrated backscatter to get either DEM-based sigma0 or DEM-based gamma0.

Comment (hjkristenson) [reply docs/guides/rtc_product_guide.md]: OK, thanks for checking that out.

File: docs/guides/rtc_product_guide.md [hunk 3]
```diff
@@ -1,30 +1,93 @@
...
+
+The two Sentinel-1 satellites each have a 12-day repeat cycle, but their orbits are offset 180 degrees so that one or the other will pass over the same location on earth every 6 days. Most areas of the earth will still only have imagery collected every 12 days, but Europe and select areas of interest are imaged with a 6-day interval, as described in the [mission observation scenario](https://sentinels.copernicus.eu/web/sentinel/missions/sentinel-1/observation-scenario).
+
+Because this is a polar-orbiting satellite constellation, areas near the poles may have a number of overlapping paths, resulting in even more frequent acquisitions with similar footprints. 
+
+The relatively short interval between acquisitions makes this SAR dataset a very useful tool for monitoring rapid or sudden landscape changes. In addition, SAR can image the earth's surface through cloud or smoke cover and does not require sunlight, so valid imagery can be collected on every pass. This is particularly useful for monitoring conditions during natural disasters such as hurricanes or wildfires, or in areas that are prone to frequent cloud cover.
+
+### SAR Distortions
```
Comment (asjohnston-asf) [inline docs/guides/rtc_product_guide.md:15]: This header might be better as `Correcting for SAR Distortions`, or something similar?  The whole page is about RTC, but there's no header or table of contents entry that calls out "here's what RTC is"

**Comment by asjohnston-asf** [comment]
Much of the existing DEM content is duplicated at https://hyp3-docs.asf.alaska.edu/dems/ .  We created that place as a shared page to call out DEMs used for all of our products (RTC, InSAR, autoRIFT).  We should update the dems/ page as well, and could consider having the RTC guide give a brief summary and link to it for the full details.

**Comment by asjohnston-asf** [comment]
Still a little bit of inconsistency with how we name the Copernicus DEM (I've been struggling to standardize, too).  From the [dataset page](https://spacedata.copernicus.eu/fr/dataset-details?articleId=394198) , the "big" dataset is the "Copernicus DEM (COP-DEM)", and the sub-dataset we're using is "GLO-30 Public".

"GLO-30 Copernicus DEM" and "Copernicus DEM GLO-30" are probably both correct, but we might stick to just "Copernicus DEM" when talking about the DEM(s) as a whole, and just "GLO-30" or "GLO-30 Public" when talking about the specific dataset.

File: docs/guides/rtc_product_guide.md (hunk 3 above)
Comment (hjkristenson) [reply docs/guides/rtc_product_guide.md:15]: @asjohnston-asf I adjusted the headers in the Terrain Correction workflow to reflect the radiometric part of the process. Does that address your concerns? Another approach would be to add a bit of a high-level description of RTC in that very first blurb under the Product Guide title.
@talogan can you take a look to make sure those headers are appropriate? I also added a reference to the DEM mapping function, as we didn't really say anywhere in the workflow description how the DEM was used in the process. Amend as suitable.

Comment (talogan) [reply docs/guides/rtc_product_guide.md:15]: The headers look good to me.

File: docs/dems.md [hunk 4]
```diff
@@ -1,49 +1,70 @@
 # Digital Elevation Models (DEMs)
```
Comment (talogan) [inline docs/dems.md:1]: i just copied the DEM section from the rtc_product_guide here.

Comment (hjkristenson) [reply docs/dems.md]: Updated the image path for the figures and added the Copernicus DEM info box

**Review by jhkennedy — APPROVED** [review]

**Pull request merged**